src/lib/djerba/plugins/supplement/body/resources/TAR.description.versions.html

```diff
@@ -8,13 +8,13 @@
 
 Shallow whole genome sequencing (sWGS) libraries were prepared using the KAPA Hyper Prep kit with DNA extracted from FFPE, cfDNA, fresh frozen tissue (for tumour samples). 
 Paired-end sequencing was performed using the ${versions.TAR_ILLUMINA_VERSION} illumina technology to a minimum target coverage of 0.1x. 
-Alignments were performed using <a href=${versions.BWAMEM_LINK}>bwa mem</a> (${versions.BWAMEM_VERSION})
+Alignments were performed using <a href=${versions.BWAMEM_LINK}>bwa mem</a> (v.0.7.17)
```

Comment:
Please keep this value in `versions.py` instead of hard-coding it in two different places. Make variables called `BWAMEM1_VERSION` and `BWAMEM2_VERSION` or similar.